src/sanitizer.js

```diff
@@ -70,6 +80,15 @@ const WHITELISTED_FORMAT_TAGS = [
 
 
 /** @const {!Array<string>} */
+const WHITELISTED_ATTRS = [
+  'fallback',
+  'href',
+  'on',
+  'placeholder',
```

Comment:
should this contain all of amp's attributes? (just curious why only this set and not, noloading, layout etc)